docs/guides/routing.rst

```diff
@@ -41,14 +41,23 @@ the request to a resource view.
         if ($subpage === 'view') {
 
             // use a view for the page logic to allow other plugins to easily change it
-            echo elgg_view_resource('blog/view', [
+            $resource = elgg_view_resource('blog/view', [
```

Comment:
We have a lot to test, like `forward()` inside this resource view, or in a subview. And `forward('', '404')`.